examples/tfengine/full.hcl

```diff
@@ -107,6 +85,42 @@ template "foundation" {
   }
 }
 
+# Central secrets project and deployment.
+# NOTE: This deployment must be deployed first before any deployments in the
+# live folder. Any non-auto filled secret data must be manually filled in by
+# entering the secret manager page in console.
+template "project_secrets" {
+  recipe_path = "{{$base}}/org/project.hcl"
+  output_path = "./live/secrets"
+  data = {
+    project = {
+      project_id = "example-secrets"
+      apis = [
+        "secretmanager.googleapis.com"
+      ]
+    }
+    resources = {
+      secrets = [
+        {
+          secret_id = "manual-sql-db-user"
+        },
+        {
+          secret_id   = "auto-sql-db-password"
+          secret_data = "$${random_password.db.result}" // Use $$ to escape.
```

Comment:
I feel like if a secret is "auto-*", it would be neat if the engine generated a "random_password" block for it. Would that be hard to add?

Reply:
That's a good idea, let me put a TODO for this.